sections/semantics-sections.include

```diff
@@ -820,10 +843,16 @@
   element is said to have the highest rank, the <code>h6</code> element has the lowest rank, and two
   elements with the same name have equal rank.
 
-  <code>h1</code>–<code>h6</code> elements must not be used to markup subheadings, subtitles,
+  <code>h2</code>–<code>h6</code> elements must not be used to markup subheadings, subtitles,
   alternative titles and taglines unless intended to be the heading for a new section or subsection.
   Instead use the markup patterns in the [[#common-idioms-without-dedicated-elements]] section of
   the specification.
+  
+  <p>The semantics of the <code>h1</code>–<code>h6</code> elements may be conveyed to users of Assistive Technology. 
+  This information can provide a hint to users as to the type of content. For example the <a>role</a> of
+   the element, which in this case is "heading" and the heading level "1" to "6", can be 
+   announced by screen reader software when a user navigates to an <code>h1</code>–<code>h6</code> element. 
```

Comment:
I would rewrite this:

"Assistive technology often announces the presence and level of a heading to users, as a hint to understand the structure of a document and construct a 'mental model' of its outline."